prosemirror/model/to_dom.py

```diff
@@ -173,22 +188,26 @@ def render_spec(
         return dom, content_dom
 
     @classmethod
-    def from_schema(cls, schema: Schema) -> "DOMSerializer":
+    def from_schema(cls, schema: Schema[str, str]) -> "DOMSerializer":
         return cls(cls.nodes_from_schema(schema), cls.marks_from_schema(schema))
 
     @classmethod
-    def nodes_from_schema(cls, schema: Schema):
+    def nodes_from_schema(
+        cls, schema: Schema[str, str]
+    ) -> Dict[str, Callable[["Node"], HTMLOutputSpec]]:
         result = gather_to_dom(schema.nodes)
         if "text" not in result:
             result["text"] = lambda node: node.text
         return result
 
     @classmethod
-    def marks_from_schema(cls, schema: Schema):
+    def marks_from_schema(
+        cls, schema: Schema[str, str]
+    ) -> Dict[str, Callable[["Mark", bool], HTMLOutputSpec]]:
         return gather_to_dom(schema.marks)
 
 
-def gather_to_dom(obj: Dict[str, Any]):
+def gather_to_dom(obj: Dict[str, Any]) -> Dict[str, Callable]:  # type: ignore
```

Comment:
What's the type: ignore here for? If it's because of `Callable` maybe `Callable[..., Any]`?